test/stressLinear.cxx

```diff
@@ -807,7 +807,7 @@ void mstress_binary_ebe_op(Int_t rsize, Int_t csize)
   ok &= VerifyMatrixValue(m1,pattern,gVerbose,EPSILON);
   if (gVerbose)
     std::cout << "  subtracting the matrix from itself" << std::endl;
-  m1 -= m1;
+  m1 -= static_cast<TMatrixD&>(m1);
```

Comment:
Could you add a comment on the same line explaining that this is intended to fix warnings.